src/MemoryTracing/MemoryTracingUtilsTest.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/substitute.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "GrpcProtos/Constants.h"
+#include "MemoryTracingUtils.h"
+
+using orbit_grpc_protos::kMissingInfo;
+using orbit_grpc_protos::SystemMemoryUsage;
+
+namespace {
+
+void CheckMemInfoParsingResult(std::optional<SystemMemoryUsage> parsing_result,
```

Comment:
I would suggest `ExpectMemInfoParsingResult` instead of `CheckMemInfoParsingResult` as with "check" we mean something specific in our codebase, while with "expect" you hint at the fact that you use `EXPECT_...` internally.

Reply:
Changed accordingly. Thanks for suggesting the name.